plugin/relation.py

```diff
@@ -0,0 +1,463 @@
+
+            for record_type in SOCIAL_FEED_RECORD_TYPES:
+                table_name = name_for_followings_relation_index(
+                    prefix=SOCIAL_FEED_TABLE_PREFIX,
+                    record_type=record_type
+                )
+
+                create_my_followees_records_index_sql = sa.text('''
+                    INSERT INTO {db_name}.{table_name} (
+                        _id,
+                        _database_id,
+                        _owner_id,
+                        _created_at,
+                        _created_by,
+                        _updated_at,
+                        _updated_by,
+                        _access,
+                        left_id,
+                        right_id,
+                        record_ref
+                    )
+                    SELECT
+                        uuid_generate_v4() as _id,
+                        '' as _database_id,
+                        :my_user_id as _owner_id,
+                        current_timestamp as _created_at,
+                        :my_user_id as _created_by,
+                        current_timestamp as _updated_at,
+                        :my_user_id as _updated_by,
+                        '[]'::jsonb as _access,
+                        :my_user_id as left_id,
+                        record_table._owner_id as right_id,
+                        record_table._id as record_ref
+                    FROM {db_name}.{record_type} record_table
+                    JOIN {db_name}.user user_table
+                    ON (
+                        record_table._owner_id = user_table._id
+                        AND COALESCE(
+                                user_table.social_feed_fanout_policy,
+                                '{default_fanout_policy}'::jsonb
+                            ) @> '{req_fanout_policy}'::jsonb
+                    )
+                    WHERE record_table._owner_id in :my_followees_ids
+                    AND NOT EXISTS (
+                        SELECT *
+                        FROM {db_name}.{table_name}
+                        WHERE left_id=:my_user_id
+                        AND right_id IN (record_table._owner_id)
+                        AND record_ref IN (record_table._id)
+                    )
+                '''.format(
+                    db_name=DB_NAME,
+                    table_name=table_name,
+                    record_type=record_type,
+                    default_fanout_policy=SOCIAL_FEED_FANOUT_POLICY_JSON_STR,
+                    req_fanout_policy='{"following": true}'
+                ))
+                conn.execute(
+                    create_my_followees_records_index_sql,
+                    my_user_id=my_user_id,
+                    my_followees_ids=my_followees_ids_tuple
+                )
+
+
+def register_remove_index_for_friends():
+    @op('social_feed:remove_index_for_friends', user_required=True)
+    def remove_index_for_friends(friends):
+        if len(friends) <= 0:
```

Comment:
since `len` must return non-negative integer and empty list evaluates to false, you can just say:
```
if not friends:
```